### ISSUE TITLE
[craftercms] Stop task doesn't always work

### ISSUE THREAD

**Issue by sumerjabri** [craftercms/craftercms#521]
Start fresh

./gradlew init
./gradlew -Penv=authoring build deploy start
./gradlew -Penv=authoring stop

Note Tomcat is still running.

**Comment by cortiz** [comment]
Solve in PR #522

**Comment by sumerjabri** [comment]
Stop will ask Tomcat to stop, but the process stays up and it causes issues after a second start (you'll see two tomcats).

**Comment by cortiz** [comment]
Fixed on PR #840

**Comment by sumerjabri** [comment]
Stop no longer works at all.

Steps to Reproduce
--------------------
- Build the latest
- Start authoring: `./gradlew -Penv=authoring start`
- Use Studio a little bit
- Stop authoring: `./gradlew -Penv=authoring stop`

```
~/code/crafter/craftercms (master ✔) ᐅ ./gradlew -Penv=authoring stop
:stop
\e[38;5;196m
 ██████╗ ██████╗   █████╗  ███████╗ ████████╗ ███████╗ ██████╗      ██████╗ ███╗   ███╗ ███████╗
██╔════╝ ██╔══██╗ ██╔══██╗ ██╔════╝ ╚══██╔══╝ ██╔════╝ ██╔══██╗    ██╔════╝ ████╗ ████║ ██╔════╝
██║      ██████╔╝ ███████║ █████╗      ██║    █████╗   ██████╔╝    ██║      ██╔████╔██║ ███████╗
██║      ██╔══██╗ ██╔══██║ ██╔══╝      ██║    ██╔══╝   ██╔══██╗    ██║      ██║╚██╔╝██║ ╚════██║
╚██████╗ ██║  ██║ ██║  ██║ ██║         ██║    ███████╗ ██║  ██║    ╚██████╗ ██║ ╚═╝ ██║ ███████║
 ╚═════╝ ╚═╝  ╚═╝ ╚═╝  ╚═╝ ╚═╝         ╚═╝    ╚══════╝ ╚═╝  ╚═╝     ╚═════╝ ╚═╝     ╚═╝ ╚══════╝
\e[0m
------------------------------------------------------------
Stopping Deployer
------------------------------------------------------------

BUILD SUCCESSFUL

Total time: 0.692 secs
```

Tomcat logs show absolutely no reaction, as if nothing happened.
